RemoveTagTool/RemoveTagTool.py

```diff
@@ -390,36 +376,36 @@ def __opt_by_name(self, opt_name):
 
     def menu_opt_handling(self, category, option_name):
         """
-        General menu option handling
+        General menu option handling.
 
         :param category: name of the category
         :type category: string
         :param option_name: name of the menu option
         :type option_name: string
         """
-        iter_ = eval("self.dbstate.db.iter_{}_handles()".format(category))
+        iter_ = eval("self.db.iter_{}_handles()".format(category))
         filter_opt = self.__opt_by_name(option_name)
         filter_ = filter_opt.get_filter()
         objects = filter_.apply(self.dbstate.db, iter_)
         self.__remove_from(objects, category)
 
     def __remove_from(self, objects, category):
         """
-        Remove the tag from objects of a selected category
+        Remove the tag from objects of a selected category.
 
         :param objects: object handles of a selected category
         :type objects: list
         :param category: name of a selected category
         :type category: string
         """
-        db = self.dbstate.db
         counter = 0
         with DbTxn(_("Remove Tag Tool"), self.db, batch=True) as self.trans:
             self.db.disable_signals()
             num = len(objects)
             self.progress.set_pass(_('Removing tags...'), num)
             for handle in objects:
-                Object = eval("db.get_{}_from_handle(handle)".format(category))
+                Object = eval("self.db.get_{}_from_handle(handle)"
```

Comment:
Consider using the database [method](https://github.com/gramps-project/gramps/blob/master/gramps/gen/db/base.py#L1436) convenience function here, or `getattr` instead of `eval`.